test/conformance/kuttl/validate/clusterpolicy/standard/apicalls/subjectaccessreview/01-manifests.yaml

```diff
@@ -0,0 +1,58 @@
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/component: admission-controller
+    app.kubernetes.io/instance: kyverno
+    app.kubernetes.io/part-of: kyverno
+  name: kyverno:subjectaccessreviews
+rules:
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - subjectaccessreviews
+  verbs:
+  - '*'
+---
+apiVersion: kyverno.io/v2beta1
+kind: ClusterPolicy
+metadata:
+  name: check-subjectaccessreview
+spec:
+  validationFailureAction: Enforce
+  background: false
+  rules:
+  - name: check-sar
+    match:
+      any:
+      - resources:
+          kinds:
+          - ConfigMap
+    context:
+      - name: subjectaccessreview
+        apiCall:
+          urlPath: /apis/authorization.k8s.io/v1/subjectaccessreviews
+          method: POST
+          data:
+          - key: kind
+            value: SubjectAccessReview
+          - key: apiVersion
+            value: authorization.k8s.io/v1
+          - key: spec
+            value:
+              resource: "namespace"
+              resourceAttributes:
+                namespace: "{{ request.namespace }}"
+                verb: "delete"
+                group: ""
+              #user: "{{ request.userInfo.username }}"
+              user: "system:serviceaccount:kyverno:kyverno-admission-controller"
+    validate:
+      message: "User is not authorized."
+      deny:
+        conditions:
+          any:
+          - key: "{{ subjectaccessreview.status.allowed }}"
+            operator: NotEquals
+            value:  true
```

Comment:
I assume in testing this was how you discovered the admission controller could delete Namespaces and logged #6947. If we're going to fix that, doing so will break this test. It may perhaps be better to create a custom ClusterRole and ClusterRoleBinding as part of this test to prevent more rework. Here's something you can adapt if you'd like:

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: chip
rules:
- apiGroups:
  - ""
  resources:
  - pods
  verbs:
  - get
  - list
  - watch
```

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: chip
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: chip
subjects:
- kind: User
  name: chip
```